### DISCUSSION TITLE
update fooocus

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#2743] [Q&A] [answered]
which is best option if i have a fully functional version of fooocus  2.1.831

it dont update automaticaly ?!?!

e:\FooocusControl>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Update failed.
unsupported URL protocol
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.1.831
Running on local URL:  http://127.0.0.1:7865

**Comment by ILikeToasters** [comment]
One option would be to download the zip file and extract to a new folder and hopefully that should update itself unless something specific to your system that is stopping it. You could then move things like config file and other things over from previous setup.

**Comment by kalle07** [comment]
i see...

Fooocus_win64_2-1-831.7z  that was my install file 6 months ago (and its still the same)
https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7z

and I would like to avoid downloading 20GB models and stuff  (i have all)    ^^

**Reply by ILikeToasters** [reply]
It would be the same version that is downloaded but hopefully it will auto update like it should. Not sure what might be stopping yours. As for the downloads you could move the files it would normally download to the appropriate folders before running it to avoid the extra downloads. Obviously if you can figure out what is blocking it from updating that would be preferable. I personally have 4 different folders with Fooocus and share the same models folders. I use a couple different forks.

**Comment by mashb1t** [comment, marked as answer]
@kalle07 you can check what the differences are by executing the command `git status` in your Fooocus folder (the one your config is in) and also `git pull` to see why it can'r pull. If there are differences you could revert the changes.
The easiest option still is, as already pointed out by @ILikeToasters, to download the zip file again and set it up from scratch.
